### DISCUSSION TITLE
github html更新问题

### DISCUSSION THREAD

**Discussion by henry-tujia** [tiddly-gittly/TidGi-Desktop#194] [Q&A]
想问一下github的上传问题

我理解tiddlywiki上传是分两种：

1. 上传整个git包含所有文件，这个设置在软件的设置中可以体现
2. 首先生成html然后，上传index.html，这个可以在Tiddly内部的设置条目中进行设置

现状：

- wiki repo为username/wiki
- html repo为username/username.github.io

其中wiki的html中的图片地址为本地地址，即拉取到本地后可以显示，但网页上无法显示；而html repo中的图片应当被改为外置地址，可以在网页上显示（不知道理解是否正确）

但是现在可以看到wiki repo的html在不断更新，而github.io repo中的html没有更新，这是什么问题呢？

PS：大大的github图片外置插件中的网址对于我的repo有错误，github的默认分支应该改为main，而不是master

**Comment by linonetwo** [comment]
用太记的话，就是上传整个git包含所有文件了，太记会定时帮你 git add push

生成 HTML 那是靠 gh action了，与太记无关，只不过默认生成的wiki模板里面会带一个这个配置，详见 .github 文件夹

然后建议把图片全都直接拖进wiki里，这样构建的时候会自动引用当前仓库里的图片。

**Comment by linonetwo** [comment]
默认分支那个还得适配一下，有空我加下。

**Comment by linonetwo** [comment]
生成html然后，上传index.html，这个我没试过。
